docs/_news/news07.md

```diff
@@ -20,14 +20,16 @@ Curious? [Learn more](remoteoverview.html)!
 
 #### New Features and Highlights for 0.7.0
 
+**Monitoring Updates**
+- You can now [inject metrics](https://github.com/codewind-resources/design-documentation/blob/master/codewindServer/metricsInjection.md) into your Node.js, Liberty, and Spring projects. 
+- Effortlessly [visualize data](https://github.com/eclipse/codewind/issues/977) like CPU usage, memory footprint, and web traffic on the included application monitoring dashboard at the click of a button. If you don't want it or need it, simply [disable it](https://github.com/eclipse/codewind/issues/1290). Codewind hosts the dashboard, so displaying the graphs won't eat into your project's resources.
```

Comment:
Correction to
>If you don't want it or need it, simply [disable it](https://github.com/eclipse/codewind/issues/1290)

There is no way to disable our app monitoring dashboard (nor would the user care, because we host it). The PR linked there refers to the ability to enable and disable metrics *injection* (the bulletpoint above).

Reply:
**Monitoring Updates**
- You can now [inject metrics](https://github.com/codewind-resources/design-documentation/blob/master/codewindServer/metricsInjection.md) into your Node.js, Liberty, and Spring projects. You can [enable or disable](https://github.com/eclipse/codewind/issues/1290) this feature with just a click.
- Effortlessly [visualize data](https://github.com/eclipse/codewind/issues/977) like CPU usage, memory footprint, and web traffic on the included application monitoring dashboard at the click of a button. Codewind hosts the dashboard, so displaying the graphs won't eat into your project's resources.